super_backend/src/main/java/com/example/demo/entities/User.java

```diff
@@ -17,16 +23,16 @@ public class User {
     Long id;
 
     @Column(nullable = false, unique = true)
-    String username;
+     String username;
```

Comment:
this is small, but remove white space before String. This file has multiple white spaces.